src/Compilers/CSharp/Portable/Binder/Binder_Constraints.cs

```diff
@@ -86,34 +86,64 @@ internal partial class Binder
         /// <summary>
         /// Bind and return a single type parameter constraint clause.
         /// </summary>
-        private TypeParameterConstraintClause BindTypeParameterConstraints(string name, SeparatedSyntaxList<TypeParameterConstraintSyntax> constraintsSyntax, DiagnosticBag diagnostics)
+        private TypeParameterConstraintClause BindTypeParameterConstraints(
```

Comment:
>BindTypeParameterConstraints [](start = 46, length = 28)

Could you please confirm that we have tests covering all new code paths in this function?